src/pages/use-less-energy/index.js

```diff
@@ -215,8 +217,9 @@ const UseLessEnergyPage = () => (
             <p className={`${ChoiceStyles.paragraph} ${Styles.paragraph}`}>
               Lastly don’t forget to{" "}
               <Link
-                to="/choose-renewable-energy#actions"
+                to="use-less-energy/#actions"
```

Comment:
this should link to /choose-renewable-energy#actions as we want the user to change page :)